dataqs/helpers.py

```diff
@@ -381,3 +381,18 @@ def __init__(self, content):
 
     def raise_for_status(self):
         pass
+
+
+def add_keywords(keyword_list, extra_keywords):
+    """Adds extra_keywords list to the keyword_list"""
+
+    # check if datetime and category already exist in the
+    # keyword list
+
+    keywords_added = [k for k in keyword_list
+                      if k.startswith('category:') or k.startswith('datetime:')]
+
+    if keywords_added:
+        return keyword_list[:len(extra_keywords)-1] + extra_keywords
```

Comment:
@jbeezley Great point. How is this https://github.com/OpenGeoscience/dataqs/pull/48/commits/4cfc7bdd16624d2623e20bac889c5a7e3377d2a4?